clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/utils/HadoopUtils.java

```diff
@@ -52,6 +54,22 @@ public static void cleanUpHDFSPath(String path, boolean recursive) {
     }
   }
 
+  /**
+   * Set Hadoop configuration from Venice properties.
+   * @param conf Configuration to be updated
+   * @param props Venice properties
+   */
+  public static void setHadoopConfigurationFromProperties(Configuration conf, VeniceProperties props) {
+    for (String key: props.keySet()) {
+      String lowerCase = key.toLowerCase();
+      if (lowerCase.startsWith(HADOOP_PREFIX)) {
```

Comment:
I'm not sure if we should use `hadoop-conf.` prefix for this as it is the prefix used for the main MR job. There is another MR job in VPJ that validates schemas of all files and builds the dictionaries. That MR job will not be able to read from S3.

Can we create a new one (maybe `hadoop-overrides.` or something better) and in the code either:
1. Extract these general overrides and copy them to `hadoop-conf.` and `hadoop-dict-build-conf.`; or
2. Where we extract configs with these two existing prefixes, we also parse extract from the general overrides prefix

Reply:
Removed the usage. These configuration overrides are no longer needed. They should be picked up from the hadoop environment.